src/chrome/content/rules/osuosl.org.xml

```diff
@@ -1,4 +1,11 @@
 <!--
+	For other Oregon State University coverage, see
+
+		- Oregon-State-University-mismatched.xml
```

Comment:
```suggestion
```

Reply:
You mean, correcting the `*-mismatched*` file name references to `*-mismatches*`? Thanks for identifying them. Done and pushed.